src/debug_menu.cpp

```diff
@@ -1314,9 +1368,36 @@ void debug()
                     g->uquit = QUIT_NOSAVED;
                 }
                 break;
-            case DEBUG_TEST_WEATHER:
+            case DEBUG_TEST_WEATHER: {
                 weather_generator weathergen;
                 weathergen.test_weather();
+            }
+                break;
+
+
+            case DEBUG_SAVE_SCREENSHOT: {
+#if defined(TILES)
+                // Take a screenshot of the viewport.
+                tilecontext.get()->save_screenshot();
+#else
+                popup(_("This binary was not compiled with tiles support."));
```

Comment:
Astyle will complain about these parens (and some more below).

Reply:
Strangely Astyle 3.0.1 or 3.1 doesn't complain...

Reply:
Do you use [.astylerc](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/.astylerc) as options for `astyle`?

Reply:
Yes I do. I also checked manually using Astyle the command line on this file but it didn't report any changes...